thoth/common/openshift.py

```diff
@@ -1528,11 +1529,15 @@ def schedule_mi_workflow(
                                        multiple entities are in form of Foo,Bar,...
         """
         workflow_id = job_id or self.generate_id("mi")
+
+        deployment_name = os.environ["THOTH_DEPLOYMENT_NAME"]
+        path = Path(deployment_name).joinpath("mi").joinpath(knowledge_path)
```

Comment:
I think this logic can be created on mi-scheduler directly and passed here with `knowledge_path`, so no need for these changes